filters/all.js

```diff
@@ -1,82 +1,75 @@
+const filter = module.exports;
```

Comment:
As i understood, version of generator which will support this is not merged to master yet.
So, it's difficult to test, but just by looking it's OK.

Reply:
I tested if generation still works, and it works
If you already have an environment where you can test if generated application works, awesome, otherwise we will just merge making sure generation works. All should be fine, I didn't touch the logic

Reply:
Hmm, i built docker image from your fork https://github.com/derberg/generator/tree/use-filters-lib and then run with one of my "test asyncapi":
```bash
 docker run --rm -it -v ../java-spring-template-derberg:/app/fork -v $pwd/asyncapi.yml:/app/asyncapi.yml -v $pwd/output:/app/output derberg-generator -o ./output asyncapi.yml /app/fork --force-write 
npm http fetch GET 200 https://registry.npmjs.org/markdown-it 119ms
npm http fetch GET 200 https://registry.npmjs.org/markdown-it/-/markdown-it-10.0.0.tgz 30ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/openapi-sampler 904ms
npm http fetch GET 200 https://registry.npmjs.org/openapi-sampler/-/openapi-sampler-1.0.0-beta.15.tgz 44ms
npm http fetch GET 200 https://registry.npmjs.org/entities 29ms
npm http fetch GET 200 https://registry.npmjs.org/entities/-/entities-2.0.0.tgz 18ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/json-pointer 813ms
npm http fetch GET 200 https://registry.npmjs.org/json-pointer/-/json-pointer-0.6.0.tgz 36ms
npm http fetch GET 200 https://registry.npmjs.org/foreach 27ms
npm http fetch GET 200 https://registry.npmjs.org/foreach/-/foreach-2.0.5.tgz 25ms
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ @asyncapi/java-spring-template@0.7.0
added 7 packages from 7 contributors in 41.233s

54 packages are looking for funding
  run `npm fund` for details

Something went wrong:
Error: Cannot find module '/app/node_modules/@asyncapi/java-spring-template/node_modules/@asyncapi/generator-filters'
Require stack:
- /app/lib/filtersRegistry.js
- /app/lib/generator.js
- /app/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at /app/lib/filtersRegistry.js:71:17
```
@derberg what i missed?

Reply:
there is a new package in the template -> https://github.com/asyncapi/java-spring-template/pull/22/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R32 so you need to make sure you have it installed locally, `npm i`

Reply:
Ok, now i recognize it. Actually we have committed `node_modules` folder in repository. I'd add it to `.gitignore` but folder itself still present. That's the first issue.
Second, i run generator from docker container exactly to avoid any manual installations with npm. In fact, later i even may don't have a `npm`, but generator worked, because of committed `node_modules` with all dependencies, i suppose.
So, two options - add new dependency to committed `node_modules` and remove it from `.gitignore`, or run `npm i` during run of generator to install all required dependencies.

Reply:
I didn't notice `node_modules` is in repo. It should not be, not a good practice. It should be in `.gitignore`. I will remove `node_modules` now, gimme few minutes

Reply:
ok done, node_modules removed in this PR and added to `.gitignore`

Reply:
OK, no `node_modules` in the repo,  but, nevertheless, the user should not make any local `npm` installations if he starts the generator from the container.

Reply:
hmmmm, generator is a docker container in this case, but template not, it is a normal npm package in local system mounted into container, that has dependencies and anyone using the package should always run `npm i` to get dependencies installed first

Reply:
Ok, i wasn't familiar about difference how generator treat local templates and templates as npm package or github. I switched to you git repo and it seems to work.
`docker run --rm -it -v $pwd/asyncapi.yml:/app/asyncapi.yml -v $pwd/output:/app/output derberg-generator -o ./output asyncapi.yml https://github.com/derberg/java-spring-template.git#use-filters-lib --force-write`

Reply:
Great stuff then! 
Wait for merge of generator.